default/stringtables/en.txt

```diff
@@ -7935,57 +7943,110 @@ Launch Bay
 
 PC_FIGHTER_BAY_DESC
 '''Launch bays are the access point for [[encyclopedia FIGHTER_TECHS]] to enter or exit a ship.
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Each launch bay allows a number of fighters to launch each combat round.  Launched fighters take one combat round to reach firing range.'''
```

Comment:
ambiguous. say fighters can attack the round after they are launched, and any later rounds until a combat ends